test/chroot_test.dart

```diff
@@ -31,58 +31,61 @@ void main() {
       );
     });
 
-    group('localBacked', () {
-      ChrootFileSystem fs;
-      io.Directory tmp;
-
-      setUp(() {
-        tmp = io.Directory.systemTemp.createTempSync('file_test_');
-        tmp = new io.Directory(tmp.resolveSymbolicLinksSync());
-        fs = new ChrootFileSystem(new LocalFileSystem(), tmp.path);
-      });
+    // LocalFileSystem is broken on Windows
+    if (!io.Platform.isWindows) {
```

Comment:
would it be better to mark the group as "skip: io.Platform.isWindows"? That way, you see during test execution that some tests were skipped.